### DISCUSSION TITLE
Can we leverage server side redirects?

### DISCUSSION THREAD

**Discussion by techtolentino** [Qiskit/qiskit.org#2659] [Ideas]
With the recent merge of the Learn page migration, we have a leftover URL that a lot resources _might_ be pointing to, that being [https://qiskit.org/textbook-beta](https://qiskit.org/textbook-beta)

If you click on that link, you'll see an HTML redirect, implemented [here](https://github.com/Qiskit/qiskit.org/blob/main/pages/textbook-beta.vue).

This discussion is to figure out a better way of doing redirects.

@korgan00 shared this resource that we could use for inspiration, doing server side redirects.
https://deltener.com/blog/nuxt-redirects/

It'd be great to hear the team's thoughts on this.

⮂

**Comment by abdonrd** [comment]
I am in favor of using server side redirects.
